ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -199,7 +199,7 @@ objects:
 
         ansible-playbook -vv \
           -e openshift_test_repo="${PREVIOUS_RPM_REPO}" \
-          -e 'oreg_url=${oreg_url}' \
+          -e oreg_url=${oreg_url} \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
```

Comment:
Double-quote these two lines.

Also, the `upgrade` container below has similar code, let's change that too.

Reply:
okay, I really hope this won't expand '${component}' this time

Reply:
Done